src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -64,6 +65,20 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
 
         #endregion
 
+        #region Cancellation support
+
+        private readonly CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
+
+        private CancellationToken cancellationToken
```

Comment:
Private property makes no sense. We can use cancellationTokenSource.Token directly.

Reply:
Good catch! Updated 😁